### PR TITLE
feat: bootstrap examples

### PR DESCRIPTION
**Pull request by xiaohuoni**
## Contributing to umi examples

Such as creating example `hello-world`.

```bash
$ mkdir -p examples/hello-world
$ yarn bootstrap:example
```

Then you will find the `README.md` and `package.json` is generated in `examples/hello-world`.

```bash
$ tree examples/hello-world
examples/hello-world
├── pages/index.tsx
├── README.md
└── package.json
```

## How to use

Execute [`@umijs/create-umi-app`](https://github.com/umijs/umi/tree/master/packages/create-umi-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:

```bash
npx @umijs/create-umi-app --example hello-world hello-world-app
# or
yarn create @umijs/umi-app --example hello-world hello-world-app
```

##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

- any feature?
- https://github.com/umijs/umi/issues/6530

-----
[View rendered CONTRIBUTING.md](https://github.com/umijs/umi/blob/feat-bootstrap-examples/CONTRIBUTING.md)
[View rendered examples/hello-world/README.md](https://github.com/umijs/umi/blob/feat-bootstrap-examples/examples/hello-world/README.md)

### CODE REVIEW

**Comment by xiaohuoni** [comment]
window 测试一直没通过，我回家用台式机跑一下看看。这个今天完不成了。

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -107,6 +107,25 @@ Running examples can be done with:
 yarn example dev examples/normal
 ```
 
+### Create new examples
+
+Such as creating example `hello-world`.
+
+```bash
+$ mkdir -p examples/hello-world
+$ yarn bootstrap:example
```
Comment (sorrycc) [inline CONTRIBUTING.md:116]:
```suggestion
$ yarn bootstrap:examples
```

File: packages/create-umi-app/src/index.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import { yargs } from '@umijs/utils';
+import download from 'load-examples';
```
[the remark this answers is not in the document]
Comment (sorrycc) [reply packages/create-umi-app/src/index.ts:2]: 也可以分开提 PR，先和 examples bootstrap 的部分。

[the remark this answers is not in the document]
Comment (xiaohuoni) [reply packages/create-umi-app/src/index.ts]: 先和 examples bootstrap 的部分

**Pull request merged**